src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridgeMeterRegistryProducer.java

```diff
@@ -54,13 +59,12 @@ CustomMeterRegistryProducer createPrometheusMeterRegistry() {
     @Override
     public String scrape() {
```

Comment:
Javadoc as ...

`Scrape Quarkus built-in meter registry for HTTP related metrics and  JMX collector registry for Kafka clients related metrics`